.github/ISSUE_TEMPLATE/template-graduation-application.md

```diff
@@ -0,0 +1,273 @@
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **Document how the project makes decisions on leadership, contribution acceptance, requests to the CNCF, and changes to governance or project goals.**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **Document how role, function-based members, or sub-teams are assigned, onboarded, and removed for specific teams (example: Security Response Committee).**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **Document complete list of current maintainers, including names, contact information, domain of responsibility, and affiliation.**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **A number of active maintainers which is appropriate to the size and scope of the project.**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **Document a complete maintainer lifecycle process (including roles, onboarding, offboarding, and emeritus status).**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **Demonstrate usage of the maintainer lifecycle with outcomes, either through the addition or replacement of maintainers as project events have required.**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **Project maintainers from at least 2 organizations that demonstrates survivability.**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **Code and Doc ownership in Github and elsewhere matches documented governance roles.**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **Document agreement that project will adopt CNCF Code of Conduct.**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **CNCF Code of Conduct is cross-linked from other governance documents.**
+
+<!-- (Project assertion goes here) -->
+
+- [ ] **All subprojects, if any, are listed.**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **If the project has subprojects: subproject leadership, contribution, maturity status documented, including add/remove process.**
+
+<!-- (Project assertion goes here) --> 
+
+## Contributors and Community
+
+Note: this section may be augmented by the completion of a Governance Review from TAG Contributor Strategy.
+
+### Suggested
+
+- [ ] **Contributor ladder with multiple roles for contributors.**
+
+<!-- (Project assertion goes here) --> 
+
+### Required
+
+- [ ] **Clearly defined and discoverable process to submit issues or changes.**
+
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **Project must have, and document, at least one public communications channel for users and/or contributors.**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **List and document all project communication channels, including subprojects (mail list/slack/etc.).  List any non-public communications channels and what their special purpose is.**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **Up-to-date public meeting schedulers and/or integration with CNCF calendar.**
```

Comment:
I don't think that having periodic meetings should be a hard requirement, as it is quite challenging for projects with timezone diversity.

Reply:
Think of it more like continuity instead.
A meeting should be public and discoverable for what's happened where, instead 'so every two weeks we..'.